### PR TITLE
Add snap scrolling

### PR DESCRIPTION
**Pull request by falbru**
Desktop (no scrolling needed):

image

Mobile (snap scrolling):

https://github.com/user-attachments/assets/31d351f4-04d0-427e-b3d2-ad6ec4042c44

Also made the footer smaller. I experimented with keeping the bottom wave, but although it looked dope it took up a lot of space. I also like that it now has fewer colors in the background (it feels less cluttered imo).

### CODE REVIEW

**Comment by falbru** [comment]
I agree that there's too much empty space. It looks a lot better on mobile, and I was planning to add more bling to the header so that it becomes taller anyways. However, I think that it still will be too long on desktop. I will try to make it smaller on desktop!

File: src/components/footer.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-const Footer = () => {
+const Footer = ({ className }: Props) => {
   return (
-    <div className="w-full flex items-center flex-col bg-lego">
-      <Image src="/images/footer.svg" alt="Footer" width={500} height={200} />
-      <div className="w-full bg-lego-muted flex flex-col items-center justify-center p-6 space-y-2">
-        <span className="text-white text-gray">Laget med 🍺 av Webkom</span>
-        <span className="text-white">© Abakus 2024</span>
-      </div>
+    <div className={`w-full ${className} text-black text-center mt-auto`}>
```
Comment (Bestem0r) [inline src/components/footer.tsx:7]: Can we have some vertical padding on the footer please? @falbru

**Review by Bestem0r — APPROVED** [review]
Looks great!

File: src/components/footer.tsx (hunk 1 above)
Comment (falbru) [reply src/components/footer.tsx:7]: NOOO

**Pull request merged**